pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -1039,6 +1039,13 @@ let
     } // optionalAttrs (versionAtLeast version "5.4" && stdenv.hostPlatform.system == "x86_64-linux") {
       CHROMEOS_LAPTOP = module;
       CHROMEOS_PSTORE = module;
+    } // optionalAttrs (versionAtLeast version "6.3" && (stdenv.hostPlatform.system == "x86_64-linux" || stdenv.hostPlatform.system == "aarch64-linux")) {
+      GCC_PLUGINS = no;
```

Comment:
Why do we drop this now, is this incompatible with Rust?

Reply:
Yes.